IndomitableIbises/Bangazon-API-Sprint-1
# Deleted duplicate virtual environment from master

## Link to Ticket
No ticket

## Description of Proposed Changes

Removed duplicate virtual env in master
Adds in a feature to ignore vs code personal settings

## Steps to Test

```sh
git fetch --all
git checkout fix_env
```

1. Once in new branch cd into first level of bangazon
1. Run ls and make sure bangenv folder is no longer there
1. Delete database and re-run migrate
1. Run server and make sure all is still working correctly

## Impacted Areas in Application

List general components of the application that this PR will affect:

*Removed duplicate env folder from master
*Adds in a feature to ignore vs code personal settings

## Mentions @username

@sirwin4 @jcarter0149 @rafcevallos 

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
